### PR TITLE
Add Dex server selector, fix selector not scrollable

### LINKED ISSUE #1009: Add Dex Server selector design update
At the Watllet selector page implement Dex Server selector section bellow Wallet selector.
Dex Server selector section will list all available servers addresses from library, and ability to create new custom Dex server save it in App storage to reuse for next session.

Screen Shot 2022-06-27 at 20 59 47

### LINKED ISSUE #1010: Wallets selector is not scrollable
Wallets selector page is not scrollable when i have 6 or add more wallets, 
Expected: user can scroll vertical on this page.
Screen Shot 2022-06-27 at 22 00 28

### PR DESCRIPTION
**Pull request by song50119**
Fix #1010 
Resolves #1009 

image

### CODE REVIEW

**Review by Sirmorrison — CHANGES_REQUESTED** [review]
The entire page scrolls which should not be.. the title and create wallet button should not scroll.

File: ui/page/dexclient/add_dex_modal.go [hunk 1]
```diff
@@ -352,3 +235,17 @@ func confirmRegisterModalDesc(dexServer *core.Exchange, selectedFeeAsset string)
 	feeAmt := formatAmountUnit(feeAsset.ID, selectedFeeAsset, feeAsset.Amt)
 	return fmt.Sprintf("Confirm DEX registration. When you submit this form, %s will be spent from your wallet to pay registration fees.", feeAmt)
 }
+
+// saveDexServer after pay the fee success save the host and cert to db.
+func (md *AddDexModal) saveDexServer(host string, cert []byte) {
```
Comment (itswisdomagain) [inline ui/page/dexclient/add_dex_modal.go:240]: Why are these details saved here after successful fee payment? Doesn't the official dex client library handle this?

**Review by itswisdomagain — CHANGES_REQUESTED** [review]
The debug setting option to reset the DEX client is no longer available. I'm unable to properly test this PR.

Also, on app load, the select wallet page also prompts to select a DEX server. Not sure what the idea for that is. Do you need to select a wallet AND a DEX server? Can you proceed by just selecting a DEX server but not selecting a wallet? Where page does that lead to?

**Comment by Sirmorrison** [comment]
66 transaction(s) indexed in total
2022-07-18 09:41:06.125 [INF] DLWL: [108] Connected block 000000094322b5a7f2d4f8aab243d11b7bad8f9e2a9d8ab4f54ab06fdaf742fd, height 958821, 0 wallet transaction(s)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x4dca74a]

goroutine 844 [running]:
github.com/planetdecred/godcr/ui/page/components.(*WalletSelector).listenForTxNotifications.func1()
	/Users/sirmorrison/Desktop/go projects/godcr-gio/ui/page/components/wallet_selector.go:399 +0x14a
created by github.com/planetdecred/godcr/ui/page/components.(*WalletSelector).listenForTxNotifications
	/Users/sirmorrison/Desktop/go projects/godcr-gio/ui/page/components/wallet_selector.go:389 +0xfc
sirmorrison@sirmorrisons-MacBook-Pro godcr-gio % 

its crashing... and you need to rebase.

**Review by Sirmorrison — APPROVED** [review]

**Review by dreacot — APPROVED** [review]
LGTM

**Pull request merged**